themes/default/content/registry/packages/azure-native/installation-configuration.md

```diff
@@ -58,9 +63,49 @@ Pick out the `<id>` from the list and run:
 $ az account set --subscription=<id>
 ```
 
-#### Authenticate using a Service Principal
+### Authenticate using OpenID Connect (OIDC)
+
```

Comment:
formatting a little odd here?

suggested text:

OIDC allows you to establish a trust relationship between Azure and another identity provider such as GitHub. Once established, your program can exchange a token issued by the identity provider (in this case, GitHub) for an Azure token. For example, using OIDC, your Pulumi program can run in GitHub Actions CI and access Azure, without storing any secrets in GitHub.